tasks/orka-init.yaml

```diff
@@ -60,7 +88,7 @@ spec:
   results:
     - name: vm-name
   steps:
-    - name: init
+    - name: initialize
```

Comment:
Same thing here, `step-init` works in Tekton v0.16.3 but not with latest version.  Changing the `name` of the `step` worked.  Got this error in latest version:  ```failed to create task run pod "run-parallel-deploy-setup": Pod "run-parallel-deploy-setup-pod" is invalid: spec.initContainers[1].name: Duplicate value: "step-init". Maybe missing or invalid Task default/orka-init```

Reply:
Looks like this has been updated in the official Tekton catalog: https://github.com/tektoncd/catalog/blob/main/task/orka-init/0.1/orka-init.yaml#L80

We should likely adopt and mirror the changes made there, since when we publish a new version we will also publish upstream to the catalog